contracts/proxy-lib/tests/common/mod.rs

```diff
@@ -35,3 +35,17 @@ pub async fn create_account_with_balance(
         .into_result()?;
     Ok(account)
 }
+
+pub fn generate_near_account_id() -> Result<String> {
+    let random_account_id: String = rand::thread_rng()
+        .sample_iter(&rand::distributions::Uniform::new_inclusive('a', 'z')) // Lowercase letters
+        .take(6)
+        .chain(
+            rand::thread_rng()
+                .sample_iter(&rand::distributions::Uniform::new_inclusive('0', '9'))
+                .take(2),
+        ) // Append digits
+        .collect();
+
+    Ok(random_account_id)
+}
```

Comment:
can we `Worker::dev_create_account` instead?

https://docs.rs/near-workspaces/latest/near_workspaces/struct.Worker.html#method.dev_create_account